packages/workflow/src/utils.ts

```diff
@@ -181,7 +182,16 @@ export const replaceCircularReferences = <T>(value: T, knownObjects = new WeakSe
 	knownObjects.add(value);
 	const copy = (Array.isArray(value) ? [] : {}) as T;
 	for (const key in value) {
-		copy[key] = replaceCircularReferences(value[key], knownObjects);
+		try {
+			copy[key] = replaceCircularReferences(value[key], knownObjects);
+		} catch (error: unknown) {
+			// Skip properties that cannot be assigned to (readonly, non-configurable, etc.)
+			LoggerProxy.error(
+				'Error while replacing circular references: ' +
+					(error instanceof Error ? error.message : 'unknown'),
+				{ error },
+			);
+		}
```

Comment:
It's not a good practice to catch all and "ignore" them. Instead we should identify the specific error, handle that, and rethrow all the rest